app/components/dashboard/DcChart.vue

```
@@ -1883,7 +1812,8 @@ const mm = {
     },
     cCond: {
       barWidth: 40,
-      colors: COL_CND
+      colors: COL_CND,
+      ordinalColors : isDark ? d3.schemeDark2 : colorbrewer.Pastel1[9]
```

ESLint

[error] Extra space after key 'ordinalColors'.

(@stylistic/key-spacing)